snapcraft/main.py

```diff
@@ -117,9 +120,22 @@ def _get_version():
 
 
 def _list_plugins():
+    plugins = []
     for importer, modname, is_package in pkgutil.iter_modules(
             snapcraft.plugins.__path__):
-        print(modname.replace('_', '-'))
+        plugins.append(modname.replace('_', '-'))
+
+    # we wrap the output depending on terminal size
+    width = MAX_CHARACTERS_WRAP
+    with suppress(OSError):
+        with suppress(subprocess.CalledProcessError):
+            # this is the only way to get current terminal size reliably
+            # without duplicating a bunch of logic
+            command = ['tput', 'cols']
```

Comment:
Isn't it? :)
I remember to have been surprised by this as well a year ago, when looking for it, and after some digging into the web, that was the only way to get this done. I just did a quick look again yesterday, and nothing changed in between